CONTRIBUTING.md

```diff
@@ -4,10 +4,9 @@ We would love to implement your contributions to this project! We simply ask tha
 
 ## Code Style
 
-This project follows the [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html).
+This project follows the [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html), and for imports, we like to put put external imports in alphabetical order first, a new line, and then local imports.
```

Comment:
This line is not grammatically correct. Can you please change it to:

```
This project follows the [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html), with the following modifications:
- imports are specified in alphabetical order
- built-in imports, external imports, and local imports are grouped together, in that order, separated by newlines
```

Reply:
Addressed in [135de20](https://github.com/OpenAdaptAI/OpenAdapt/pull/305/commits/135de2083414fc3f1e23ed82aa49f662f0a84b65)